src/d3a/models/area/stats.py

```diff
@@ -90,3 +94,26 @@ def cheapest_offers(self):
         for market in self._markets.markets.values():
             cheapest_offers.extend(market.sorted_offers[0:1])
         return cheapest_offers
+
+    def min_max_avg_rate_market(self, time_slot):
+        out_dict = {"min_trade_rate": None,
+                    "max_trade_rate": None,
+                    "avg_trade_rate": None}
+        for market in self._markets.all_spot_markets:
+            if market.time_slot == time_slot and len(market.trades) > 0:
+                trade_rates = [trade.offer.price/trade.offer.energy for trade in market.trades]
+                out_dict["min_trade_rate"] = round(min(trade_rates), 6)
```

Comment:
If these are calculated only for the external API, I would recommend either using our default precision constant from constants.py, or not round it at all.

Reply:
OK, using DEFAULT_PRECISION then. Thanks!

Reply:
We also have a util function `limit_float_precision` that might be better to maintain code uniformity.